extensions/aaudio-cffi.lisp

```diff
@@ -154,19 +154,19 @@
   (:media 1)
   (:voice-communication 2)
   (:voice-communication-signalling 3)
-  (:usage-album 4)
-  (:usage-notification 5)
-  (:usage-notification-ringtone 6)
-  (:usage-notification-event 10)
-  (:usage-assistance-accessibility 11)
-  (:usage-assistance-navigation-guidance 12)
-  (:usage-assistance-sonification 13)
-  (:usage-game 14)
-  (:usage-assistant 16)
-  (:system-usage-emergency 1000)
-  (:system-usage-safety 1001)
-  (:system-usage-vehicle-status 1002)
-  (:system-usage-announcement 1003))
```

Comment:
Why? I removed the prefixes for a reason.

Reply:
They were not removed? Starting from `:usage-album` (which seems to be `:alarm` istead)

Reply:
I think I'm confused with all of your patches because github is messing up the diffs somehow. Sorry for all the noise.